unit-tests/src/org/commcare/android/tests/caselist/EntityListCalloutDataTest.java

```diff
@@ -112,9 +115,7 @@ public void testCalloutResultWithNoColumnTest() {
 
     private void performFingerprintCallout() {
         // make entity list callout to 'fingerprint identification'
-        ImageButton calloutButton =
-                (ImageButton)entitySelectActivity.findViewById(R.id.barcodeButton);
-        calloutButton.performClick();
+        entitySelectActivity.barcodeScanOnClickListener.onClick(null);
```

Comment:
Is there a particular reason for this change? Would be nice to leave `barcodeScanOnClickListener` as `private`